### PR TITLE
Minor edit

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### What does this implement/fix? Explain your changes.
+ Installation instruction modified
+ Test system optimized
+ Different ways of the `matplotlib` import unified
+ doctests' issues fixed

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -34,33 +38,33 @@ jobs:
     - name: Test with pytest
       run: |
         python -m pytest --ignore-glob="*nft_upload_test.py" test --cov=samila --cov-report=term
-      if: matrix.python-version != 3.5
+      if: matrix.python-version != env.TEST_PYTHON_VERSION
```
Comment (sepandhaghighi) [inline .github/workflows/test.yml:41]: This line is an exception for version 3.5, remove it (it seems now it works fine with this version)

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: samila/functions.py [hunk 2]
```diff
@@ -9,7 +9,6 @@
 import json
 import random
 import matplotlib
-from matplotlib import cm
 from matplotlib.colors import ListedColormap
```
Comment (sepandhaghighi) [inline samila/functions.py:12]: Also about this line

File: .github/workflows/test.yml (hunk 1 above)
Comment (sadrasabouri) [reply .github/workflows/test.yml:41]: Aha. You mean removing the if or reverting the commit?

Comment (sadrasabouri) [reply .github/workflows/test.yml]: It's better to remove the line :smile:

Comment (sepandhaghighi) [reply .github/workflows/test.yml]: Sorry for my late response ⏲️ 
Yeah, I meant to delete it 💯

**Review by sepandhaghighi — APPROVED** [review]
@sadrasabouri Well done 🚀

**Pull request merged**